app/components/files/file.tsx

```diff
@@ -13,6 +13,7 @@ import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import DocumentFile, {DocumentFileRef} from './document_file';
```

Comment:
move this 'file' to its own component

Reply:
https://mattermost.atlassian.net/browse/MM-45614

Reply:
I would prefer having this one sorted out before pushing it into Gekidou - let's try to keep Gekidou as clean as possible.